lib/masamune/schema/column.rb

```diff
@@ -456,6 +456,14 @@ def sequence_id
       "#{reference_name}_seq" if type == :sequence
     end
 
+    def required_value?
+      if reference
+        !(reference.null || reference.default)
+      else
+        surrogate_key || natural_key
+      end
```

Comment:
only keys are required, referenced values are only required if the reference cannot be null or there is no default value defined